probatus/metric_volatility/volatility.py

```diff
@@ -489,7 +519,7 @@ class BootstrappedVolatility(TrainTestVolatility):
     from sklearn.ensemble import RandomForestClassifier
     from probatus.metric_volatility import BootstrappedVolatility
     X, y = make_classification(n_features=4)
-    RandomForestClassifier()
+    clf = RandomForestClassifier()
```

Comment:
Good catch. I can show you how to write unit tests to make sure the code snippets in the docstring actually work. Feel free to plan 30min call